packages/opentelemetry/src/utils/spanData.ts

```diff
@@ -50,3 +51,13 @@ export function setSpanMetadata(span: AbstractSpan, metadata: Partial<Transactio
 export function getSpanMetadata(span: AbstractSpan): Partial<TransactionMetadata> | undefined {
   return SpanMetadata.get(span);
 }
+
+/** Set the Sentry scope to be used for finishing a given OTEL span. */
+export function setSpanFinishScope(span: AbstractSpan, scope: Scope): void {
+  SpanFinishScope.set(span, scope);
+}
+
+/** Get the Sentry scope to use for finishing an OTEL span. */
+export function getSpanFinishScope(span: AbstractSpan): Scope | undefined {
+  return SpanFinishScope.get(span);
+}
```

Comment:
I wonder if we can just ditch these getter/setter functions and just export the maps instead...

Reply:
I kind of like that this does not leak the implementation detail (=it stores in a weakmap), so we can potentially change this if we want!